src/components/bulkConcepts/container/BulkConceptsPage.jsx

```diff
@@ -121,7 +161,9 @@ export class BulkConceptsPage extends Component {
               preview={preview}
               previewConcept={this.props.previewConcept}
               addConcept={this.props.addConcept}
+              handleNextPage={this.handleNextPage}
```

Comment:
@Mrshinezee, please destructure props here.

Reply:
done

Reply:
That's good!